AEXS/aex-141.md

```diff
@@ -105,9 +109,21 @@ entrypoint metadata(token_id: int) : option(metadata)
 | data | option(metadata) |
 
 The `metadata` type to use depends on the `metadata_type` defined in the contract:
-- for `URL`, `IPFS` and `OBJECT_ID` use `MetadataIdentifier`
+- for `URL` and `OBJECT_ID` use `MetadataIdentifier`
```

Comment:
no explanation here what metadata type is supposed to be used for what content

Reply:
agree, will add

Reply:
updated: https://github.com/aeternity/AEXs/commit/07261c056b5b6003749283b21a378a0a7ba7e194

Reply:
updated again: https://github.com/aeternity/AEXs/commit/a40d2842551dbd0ba4176b8bae90a01bae8f4b75

I think now it's good. but looking forward to your comment :)

Reply:
looks good